_episodes/06-func.md

```diff
@@ -804,6 +804,51 @@ readable code!
 > {: .solution}
 {: .challenge}
 
+> ## Return versus print
+>
+> Note that `return` and `print` are not interchangeable.
+> `print` is a build-in function, it sends output to the screen.
```

Comment:
Say:

> `print` is a function that _prints_ data to the screen.

Reply:
Next, would you indicate that 'return' is not like print in that it does not send output to the screen?